packages/localize-tools/testdata/build-transform-xliff/goldens/tsout/es-419/foo.js

```diff
@@ -57,6 +57,9 @@ html`&lt;Hola<b>&lt;Mundo &amp; Amigos&gt;</b>!&gt;`;
 // Expressions as attribute values should stay as expressions
 html`Hello <b foo=${'World'}>World</b>`;
 html`Hello <b foo=${`Mundo`}>World</b>`;
-html`<b foo=${'Hello'}>Hello</b><b bar=${`Mundo`}>World</b>`;
+html`<b foo=${`Hola`}>Hello</b><b bar=${`Mundo`}>World</b>`;
```

Comment:
This diff is technically unrelated to the changes in the PR but just due to a missing translation target that I noticed from the test file.